src/data/base/creditcards/creditcards.rs

```diff
@@ -0,0 +1,98 @@
+use wasm_bindgen::prelude::*;
+use crate::utils::seeder;
+use rand::Rng;
+use std::iter;
+use chrono::{Datelike, Utc};
+
+#[wasm_bindgen]
+pub fn get_cc_number() -> String {
```

Comment:
Change the function names to the format of `credit_card_{number}` to match the rest of the library function names.

Reply:
Changed function names, please check.